===================================================================
RCS file: /horde/cvs/framework/iCalendar/iCalendar/vfreebusy.php,v
retrieving revision 1.19
retrieving revision 1.20
diff -p --unified=3 -r1.19 -r1.20
--- lib/Horde/iCalendar/vfreebusy.php	2005/04/08 14:08:47	1.19
+++ lib/Horde/iCalendar/vfreebusy.php	2005/04/08 16:52:25	1.20
@@ -34,7 +34,7 @@ class Horde_iCalendar_vfreebusy extends
         // Do something with all the busy periods.
         foreach ($this->_attributes as $key => $attribute) {
             if ($attribute['name'] == 'FREEBUSY') {
-                foreach ($attribute['value'] as $value) {
+                foreach ($attribute['values'] as $value) {
                     if (array_key_exists('duration', $attribute['value'])) {
                        $this->addBusyPeriod('BUSY', $value['start'], null, $value['duration']);
                     } else {
